sdk/metricsadvisor/azure-ai-metricsadvisor/src/samples/java/com/azure/ai/metricsadvisor/administration/AnomalyDetectionConfigurationAsyncSample.java

```diff
@@ -168,17 +168,29 @@ private static void printDetectionConfiguration(AnomalyDetectionConfiguration de
                 .getSuppressCondition().getMinRatio());
 
         System.out.printf("- Hard Threshold Condition:%n");
-        System.out.printf(" - Lower bound: %s%n",
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getLowerBound());
-        System.out.printf(" - Upper bound: %s%n",
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getUpperBound());
-        System.out.printf(" - Suppress conditions: minimum number: %s; minimum ratio: %s%n",
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getSuppressCondition().getMinNumber(),
-            wholeSeriesDetectionCondition.getHardThresholdCondition()
-                .getSuppressCondition().getMinRatio());
+        if ( wholeSeriesDetectionCondition.getHardThresholdCondition() != null ) {
+            System.out.printf(" - Lower bound: %s%n",
+                wholeSeriesDetectionCondition.getHardThresholdCondition()
+                    .getLowerBound());
+            System.out.printf(" - Upper bound: %s%n",
+                wholeSeriesDetectionCondition.getHardThresholdCondition()
+                    .getUpperBound());
+            if ( wholeSeriesDetectionCondition.getHardThresholdCondition().getSuppressCondition() != null ) {
+                System.out.printf(" - Suppress conditions: minimum number: %s; minimum ratio: %s%n",
+                    wholeSeriesDetectionCondition.getHardThresholdCondition()
+                        .getSuppressCondition().getMinNumber(),
+                    wholeSeriesDetectionCondition.getHardThresholdCondition()
+                        .getSuppressCondition().getMinRatio());
+            } else {
+                System.out.printf(" - Suppress conditions: minimum number: %s; minimum ratio: %s%n",
+                    null,null);
+            }
+        } else {
+            System.out.printf(" - Lower bound: %s%n",null);
+            System.out.printf(" - Upper bound: %s%n",null);
+            System.out.printf(" - Suppress conditions: minimum number: %s; minimum ratio: %s%n",
+                null,null);
```

Comment:
We could skip printing if it is null I think.

Reply:
Skip printing when the return value is null